Makefile

```diff
@@ -14,7 +14,9 @@ lint:
 
 .PHONY: test
 coverage="false"
-test: deepcopy buildergen
+# temporary disable buildergen due
+# in github.com/galgotech/builder-gen@v0.0.0-20231121195112-e44977c5b75f): go.mod:3: invalid go version '1.20.11': must match format 1.23
```

Comment:
Once it is fixed we can revert this change back.

Reply:
Changed version to 1.19.

If necessary, change `hack/builder-gen.sh`
```patch
diff --git a/hack/builder-gen.sh b/hack/builder-gen.sh
index 083b187..c79a270 100755
--- a/hack/builder-gen.sh
+++ b/hack/builder-gen.sh
@@ -32,7 +32,7 @@ shift 1
   # make sure your GOPATH env is properly set.
   # it will go under $GOPATH/bin
   cd "$(dirname "${0}")"
-  GO111MODULE=on go install github.com/galgotech/builder-gen@latest
+  GO111MODULE=on go install github.com/galgotech/builder-gen@v0.0.1
 )
```